website/metadata.yaml

```
@@ -0,0 +1,66 @@
+pypsa-eur-sec-2-lv1-3h:
+  - Kind of model: PyPSA-Eur
+    Type of study: Infrastructure
+    Sector: Sector coupled
+    Time resolution: 3 hourly X 1 year
+    Spatial resolution: Country level (IT, 2 nodes)
+    Transmission expansion: '-'
+    Unit commitment: '-'
+    Specifications: Linear, lot of variables, strongly intermeshed constraints
+    Solution time: Highs':' 3356 s
+    '': Constraints':' 393568, Variables':'' 390692, Free variables':' 1431
```

```suggestion
  Kind of model: PyPSA-Eur
  Type of study: Infrastructure
  Sector: Sector coupled
  Time resolution: 3 hourly X 1 year
  Spatial resolution: Country level (IT, 2 nodes)
  Transmission expansion: '-'
  Unit commitment: '-'
  Specifications: Linear, lot of variables, strongly intermeshed constraints
  Solution time: Highs':' 3356 s
  '': Constraints':' 393568, Variables':'' 390692, Free variables':' 1431
```
We don't need each benchmark to be a list of dict. It can just be a dict, as per my suggestion above. I think that will be more intuitive / easier to read for the modellers who contribute benchmarks.

If you need it to be a list in your code, you can change line 37 of `benchmark.py` accordingly.